### PR TITLE
feat: add more variables for BTA & add correction file

### PR DESCRIPTION
**Pull request by Ming-Yan**
- add JEC for Summer22 (preliminary JEC from Winter22 campaign)
- fixes for BTA workflows
- add JERC and __init__.py
- add more info for example.py
- remove redundant files

### CODE REVIEW

**Comment by demuller** [comment]
Hi @Ming-Yan , thanks a lot for your PR! will start the review now

One question I already have: what did change for the JERC pickle file of the `2018_UL` campaign? (some change happened according to [link](https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/55/files#diff-0353b6c10b9de7817caa9fa92c29c31737d253bfa8e48dc9af242f8e2c33b01d))

**Review by demuller — COMMENTED** [review]
Hi @Ming-Yan looks good to me, I have one minor comment about the .root files in the data/JME folder. 

Could you also remove the 2 tar.gz files for the Summer22EERun3 JECs introduced in your last PR? E.g. [this one](https://github.com/cms-btv-pog/BTVNanoCommissioning/blob/master/src/BTVNanoCommissioning/data/JME/Summer22EERun3/Summer22EEPrompt22_RunF_V1_DATA.tar.gz)? I think there is no need to store tar.gz files in the repo

**Comment by demuller** [comment]
Also, the `Summer22Run3` campaign is not defined yet [here](https://github.com/cms-btv-pog/BTVNanoCommissioning/blob/master/src/BTVNanoCommissioning/utils/compile_jec.py#L263), but this can be done once we have the actual JERC files for that campaign

File: src/BTVNanoCommissioning/data/JME/Summer22Run3/Winter22Run3_RunCD_v1.root
[the remark this answers is not in the document]
Comment (Ming-Yan) [reply src/BTVNanoCommissioning/data/JME/Summer22Run3/Winter22Run3_RunCD_v1.root:1]: These are jet veto files, as it is for data only, I think we can keep it just in Winter22Run3 directory

**Comment by Ming-Yan** [comment]
Hi @demuller , thanks for the review!

>One question I already have: what did change for the JERC pickle file of the 2018_UL campaign? (some change happened according to [link](https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/55/files#diff-0353b6c10b9de7817caa9fa92c29c31737d253bfa8e48dc9af242f8e2c33b01d)) 

The changes for jec pickle file is recompiled with `python 3.10`( originally the files is compiled with 3.8) which requires recompiled for almost everyone.

> Also, the Summer22Run3 campaign is not defined yet [here](https://github.com/cms-btv-pog/BTVNanoCommissioning/blob/master/src/BTVNanoCommissioning/utils/compile_jec.py#L263), but this can be done once we have the actual JERC files for that campaign

This is done in purpose, since we don't have updated JEC file ready, the current suggestion written in [`AK4_parameter.py`](https://github.com/cms-btv-pog/BTVNanoCommissioning/blob/master/src/BTVNanoCommissioning/utils/AK4_parameters.py#L42-L45), this take the old file, we just update once they have it

**Review by demuller — APPROVED** [review]
Hi @Ming-Yan thanks again for the PR, I will merge it now :)

**Pull request merged**